src/SprykerEco/Zed/PunchoutCatalogs/PunchoutCatalogsDependencyProvider.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Zed\PunchoutCatalogs;
+
+use Orm\Zed\PunchoutCatalog\Persistence\PgwPunchoutCatalogConnectionQuery;
+use Spryker\Zed\Kernel\AbstractBundleDependencyProvider;
+use Spryker\Zed\Kernel\Container;
+use SprykerEco\Zed\PunchoutCatalogs\Dependency\Service\PunchoutCatalogsToUtilDateTimeServiceBridge;
+
+/**
+ * @method \SprykerEco\Zed\PunchoutCatalogs\PunchoutCatalogsConfig getConfig()
+ */
+class PunchoutCatalogsDependencyProvider extends AbstractBundleDependencyProvider
+{
+    public const PROPEL_QUERY_PUNCHOUT_CATALOG_CONNECTION = 'QUERY_CONTAINER_CUSTOMER';
+    public const SERVICE_UTIL_DATE_TIME = 'SERVICE_UTIL_DATE_TIME';
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    public function provideCommunicationLayerDependencies(Container $container): Container
+    {
+        $container = $this->addUtilDateTimeService($container);
+        $container = $this->addPunchoutCatalogConnectionPropelQuery($container);
+
+        return $container;
+    }
+
+    /**
+     * @param \Spryker\Zed\Kernel\Container $container
+     *
+     * @return \Spryker\Zed\Kernel\Container
+     */
+    protected function addUtilDateTimeService(Container $container): Container
+    {
+        $container->set(static::SERVICE_UTIL_DATE_TIME, function (Container $container) {
+            return new PunchoutCatalogsToUtilDateTimeServiceBridge($container->getLocator()->utilDateTime()->service());
```

Comment:
Please move `$container->getLocator()->utilDateTime()->service()` to the separate line.